runtime/doc/msgpack_rpc.txt

```diff
@@ -413,5 +417,30 @@ of update.
 ["popupmenu_hide"]
 	The popupmenu is hidden.
 
+["cmdline_enter"]
+	Enter the cmdline.
+
+["cmdline_leave"]
+	Leave the cmdline.
+
+["cmdline_firstc", firstc]
+        The first character of the command, which could be : / ? etc. With
+	the firstc, you know wheither it's a command or a search.
+
```

Comment:
Merging with show into a single `["cmdline_show", content, pos, firstc, prompt]` makes sense but I still think there should be a separate `cmdline_pos`, to easier avoid unnecessary redraws and it would also make the api consistent with popupmenu.